gfx/map/map_object_data/generated/steppe_bush_01_generator.txt

```diff
@@ -5,7 +5,7 @@
 	generated_content=yes
 	layer="grass_layer"
 	pdxmesh="steppe_bush_01_mesh"
```

Comment:
The shore looks kinda flat. You may notice there were bitches and cliffs on the WoW map.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132693960-c98a98f6-51a6-46f9-9df6-0b0816f5fb36.png)
![image](https://user-images.githubusercontent.com/46384992/132694347-fc4ba9ac-26bb-4be0-b08c-9546166a3c99.png)

</details>

Reply:
_Aye. The current edits to Azshara are more meant to satisfy the inaccuracies listed in #342. As mentioned, I would like to do a full revamp as part of a separate issue in the future._